<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FundTableDialog</class>
 <widget class="QDialog" name="FundTableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QTableWidget" name="fundTable">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>新建行</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>新建行</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>新建行</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>新建行</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>新建列</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>新建列</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>新建列</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>新建列</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>新建列</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="okBtn">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="moreBtn">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>更多</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="refreshBtn">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>刷新</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okBtn</sender>
   <signal>clicked()</signal>
   <receiver>FundTableDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>475</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
